src/components/Contacts/ContactFlow/ContactFlowSetup/Header/ResetToDefault/ResetToDefaultModal.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  DefaultTypeEnum,
+  getDefaultFlowOptions,
+} from 'src/components/Contacts/ContactFlow/contactFlowDefaultOptions';
+import {
+  CancelButton,
+  SubmitButton,
+} from 'src/components/common/Modal/ActionButtons/ActionButtons';
+import Modal from 'src/components/common/Modal/Modal';
+import { useContactPartnershipStatuses } from 'src/hooks/useContactPartnershipStatuses';
+import { ContactFlowOption } from '../../../ContactFlow';
+
+interface ResetToDefaultModalProps {
+  updateOptions: (options: ContactFlowOption[]) => Promise<void>;
+  handleClose: () => void;
+  resetColumnsMessage: string;
+}
+
+const MassActionsAddToAppealSchema = yup.object({
+  resetToDefaultType: yup.string().required(),
+});
+
+export const ResetToDefaultModal: React.FC<ResetToDefaultModalProps> = ({
+  handleClose,
+  updateOptions,
+  resetColumnsMessage,
+}) => {
+  const { t } = useTranslation();
+  const { contactStatuses } = useContactPartnershipStatuses();
+  const { enqueueSnackbar } = useSnackbar();
+  const [updating, setUpdating] = useState(false);
+
+  const handleOnSubmit = async (values: { resetToDefaultType: string }) => {
+    const defaultValues = getDefaultFlowOptions(
+      t,
+      contactStatuses,
+      values.resetToDefaultType as DefaultTypeEnum,
+    );
+
+    await updateOptions(defaultValues);
+    enqueueSnackbar(resetColumnsMessage, {
+      variant: 'success',
+    });
+    handleClose();
+  };
+
+  return (
+    <Modal
+      title={t('Reset to Defaults')}
+      isOpen={true}
+      handleClose={handleClose}
+    >
+      <Formik
+        initialValues={{
+          resetToDefaultType: 'US',
+        }}
+        onSubmit={(values) => {
+          setUpdating(true);
+          handleOnSubmit(values);
+        }}
+        validationSchema={MassActionsAddToAppealSchema}
+      >
+        {({
+          values: { resetToDefaultType },
+          setFieldValue,
+          handleSubmit,
+          isSubmitting,
+          isValid,
+        }): ReactElement => (
+          <form
+            onSubmit={handleSubmit}
+            noValidate
+            data-testid="CreateAppealModal"
+          >
+            <DialogContent dividers>
+              <FormControl fullWidth>
+                <Typography>
+                  {t('Pick a list to use for your newsletter')}
+                </Typography>
+                <Select
+                  name="resetToDefaultType"
+                  value={resetToDefaultType}
+                  onChange={(e) =>
+                    setFieldValue('resetToDefaultType', e.target.value)
+                  }
+                  style={{
+                    width: '100%',
+                    marginBottom: '15px',
+                  }}
+                >
+                  <MenuItem key={DefaultTypeEnum.Us} value={DefaultTypeEnum.Us}>
+                    {t('Reset columns to US (MPD) defaults')}
+                  </MenuItem>
+                  <MenuItem
+                    key={DefaultTypeEnum.Global}
```

Comment:
I don't think the `key` prop is needed because we aren't creating these menu items in a loop.

```suggestion
                  <MenuItem value={DefaultTypeEnum.Us}>
                    {t('Reset columns to US (MPD) defaults')}
                  </MenuItem>
                  <MenuItem
```